docs/posts/snowflake-io/index.qmd

```diff
@@ -0,0 +1,134 @@
+---
+title: "Icy IO: loading local files with Snowflake"
+author: "Phillip Cloud"
+date: "2023-08-31"
+categories:
+    - blog
+    - snowflake
+    - io
+    - productivity
+code-annotations: hover
+---
+## Loading files without Ibis
+
+{{< video https://www.youtube.com/embed/KFz4pbCqtQU >}}
+
+It's challenging to load local files into Snowflake from Python.
+
+Multiple steps are required, and while [those steps are
```

Comment:
I would just get rid of this and have from above

"It can be challenging to load local files into Snowflake from Python. Here's how to load a CSV file into Snowflake without Ibis:"

Reply:
Fixed!